src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -55,8 +52,7 @@ public MineColoniesAchievement(final String id, final String name, final int off
      * @param itemStackIcon The icon to show
      * @param parent        the parent achievement
      */
-    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final ItemStack itemStackIcon, final Achievement parent)
-    {
+    public MineColoniesAchievement(final String id, final String name, final int offsetX, final int offsetY, final ItemStack itemStackIcon, final Achievement parent) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)